internal/cltest/factories.go

```diff
@@ -265,26 +265,18 @@ func NewRunLog(
 }
 
 // StringToVersionedLogData encodes a string to the log data field.
-func StringToVersionedLogData(internalID, str string) hexutil.Bytes {
-	id := StringToHash(internalID).Hex()
-	j := JSONFromString(str)
-	cbor, err := j.CBOR()
-	mustNotErr(err)
-	length := len(cbor)
-	lenHex := utils.RemoveHexPrefix(hexutil.EncodeUint64(uint64(length)))
-	if len(lenHex) < 64 {
-		lenHex = strings.Repeat("0", 64-len(lenHex)) + lenHex
-	}
+func StringToVersionedLogData(internalID, str string) []byte {
+	buf := bytes.NewBuffer(hexutil.MustDecode(StringToHash(internalID).Hex()))
```

Comment:
I believe you can shorten this to `buf := bytes.NewBuffer(StringToHash(internalID).Bytes())`